src/shared/varlink-io.systemd.Repart.c

```diff
@@ -4,6 +4,60 @@
 
 #include "varlink-io.systemd.Repart.h"
 
+static SD_VARLINK_DEFINE_ENUM_TYPE(
+                ProgressPhase,
+                SD_VARLINK_DEFINE_ENUM_VALUE(loading_definitions),
+                SD_VARLINK_DEFINE_ENUM_VALUE(loading_table),
+                SD_VARLINK_DEFINE_ENUM_VALUE(opening_copy_block_sources),
+                SD_VARLINK_DEFINE_ENUM_VALUE(acquiring_partition_labels),
+                SD_VARLINK_DEFINE_ENUM_VALUE(minimizing),
+                SD_VARLINK_DEFINE_ENUM_VALUE(placing),
+                SD_VARLINK_DEFINE_ENUM_VALUE(wiping_disk),
+                SD_VARLINK_DEFINE_ENUM_VALUE(wiping_partition),
+                SD_VARLINK_DEFINE_ENUM_VALUE(copying_partition),
+                SD_VARLINK_DEFINE_ENUM_VALUE(formatting_partition),
+                SD_VARLINK_DEFINE_ENUM_VALUE(adjusting_partition),
+                SD_VARLINK_DEFINE_ENUM_VALUE(writing_table),
+                SD_VARLINK_DEFINE_ENUM_VALUE(rereading_table));
+
+static SD_VARLINK_DEFINE_ENUM_TYPE(
+                EmptyMode,
+                SD_VARLINK_FIELD_COMMENT("Refuse to operate on disks without an existing partition table"),
+                SD_VARLINK_DEFINE_ENUM_VALUE(refuse),
+                SD_VARLINK_FIELD_COMMENT("Create a new partition table if one doesn't already exist on disk"),
+                SD_VARLINK_DEFINE_ENUM_VALUE(allow),
+                SD_VARLINK_FIELD_COMMENT("Refuse to operate on disks with an existing partition table, and create a new table if none exists"),
+                SD_VARLINK_DEFINE_ENUM_VALUE(require),
+                SD_VARLINK_FIELD_COMMENT("Always create a new partition table, potentially overwriting an existing table"),
+                SD_VARLINK_DEFINE_ENUM_VALUE(force));
+
+static SD_VARLINK_DEFINE_METHOD(
+                Run,
```

Comment:
missing SD_VARLINK_SUPPORTS_MORE

Reply:
hm... probably SD_VARLINK_REQUIRES_MORE is a better fit? and require the flag in the handler